packages/extensions/openapi-to-cadl/package.json

```diff
@@ -46,7 +46,6 @@
     "@cadl-lang/prettier-plugin-cadl": "^0.5.17",
```

Comment:
Need to make a changelog`rush change` so we can release a new version. O instead bump the version in pacakge.json

Reply:
I'll wait for Jose to weigh-in. Maybe there's a reason to have cadl-python as a dependency but I suspect not. It's not an urgent PR.

Reply:
I think I used it for testing and forgot to remove it. Thanks for sending out the change